misirlou/urls.py

```diff
@@ -20,7 +20,7 @@
 urlpatterns = [
     url(r'^admin/', include(admin.site.urls)),
     url(r'^$', views.RootView.as_view(), name='api-root'),
-    url('^about/$', views.AboutView.as_view(), name='about'),
+    url('^about/$', views.RootView.as_view(), name='about'),
```

Comment:
You can remove the second URL parameter and collapse it to a single matcher, I think.

`url(r'^$|^about/$', views.RootView.as_view(), name='api-root')`